lib/cli/lib/helpers.js

```diff
@@ -1,11 +1,36 @@
 import path from 'path';
 import fs from 'fs';
 import chalk from 'chalk';
+import latestVersion from 'latest-version';
+import { sync as spawnSync } from 'cross-spawn';
+import { gt } from 'semver';
 
-const spawnSync = require('cross-spawn').sync;
+import { version, devDependencies } from '../package.json';
 
 const logger = console;
 
+export async function getVersion(packageName) {
+  const current = packageName === '@storybook/cli' ? version : devDependencies[packageName];
```

Comment:
Should we check that `packageName.match('@storybook')` or something? Otherwise we may apply this logic to other things this package depends on.

Reply:
Sounds reasonable. I think it should be `/storybook/.test(packageName)` though, as there are plans to use `storybook` package name (@ndelangen has it)